api/compute/v1alpha1/machine_types.go

```diff
@@ -23,6 +23,7 @@ type MachineSpec struct {
 	// Power is the desired machine power state.
 	// Defaults to PowerOn.
 	Power Power `json:"power,omitempty"`
+	// Deprecated: Use LocalDisk to provide a bootable disk
 	// Image is the optional URL providing the operating system image of the machine.
 	// +optional
 	Image string `json:"image,omitempty"`
```

Comment:
When we introduce now the `localDisk` volume source where the user can specify an OS image, what should happen to the `machine.spec.image`?